ds-ml/weather/weather-pipeline.py

```
@@ -0,0 +1,106 @@
+            'temperature_2m_max',
+            'temperature_2m_min', 
+            'temperature_2m_mean',
+            'precipitation_sum',
+            'rain_sum',
+            'wind_speed_10m_max',
+            'weather_code'
+        ],
+        'timezone': 'Europe/Dublin'
+    }
+    
+    response = requests.get(url, params=params)
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)